packages/react-reconciler/src/__tests__/ReactOffscreen-test.js

```diff
@@ -98,8 +98,16 @@ describe('ReactOffscreen', () => {
           <Text text="Outside" />
         </>,
       );
-      // Should not defer the hidden tree
-      expect(Scheduler).toFlushUntilNextPaint(['A', 'Outside']);
+      if (gate(flags => flags.enableDiscreteEventMicroTasks)) {
+        // Flush microtasks.
+        await null;
```

Comment:
I just remembered we actually do have an API that does this for us now — `flushSync` :D